components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -1048,7 +1053,8 @@ TEST_F(PublisherInfoDatabaseTest, DeleteActivityInfo) {
   ledger::PublisherInfoList list;
   ledger::ActivityInfoFilter filter;
   filter.excluded = ledger::EXCLUDE_FILTER::FILTER_ALL;
-  EXPECT_TRUE(publisher_info_database_->GetActivityList(0, 0, filter, &list));
+  EXPECT_TRUE(publisher_info_database_->GetActivityList(0, 0,
+      filter.Clone(), &list));
```

Comment:
`filter.Clone()` should use `std::move(filter)` as `filter` is no longer used

Reply:
fixed